pkg/apiserver/proxy.go

```diff
@@ -151,6 +151,8 @@ func (r *ProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 	httpCode = http.StatusOK
 	newReq.Header = req.Header
+	newReq.ContentLength = req.ContentLength
+	newReq.TransferEncoding = req.TransferEncoding
```

Comment:
Thanks @liggitt. I'll take a look at the issues you mentioned tomorrow or Friday.